### ISSUE TITLE
User config screen

### ISSUE THREAD

**Issue by jonganc** [Data4Democracy/ati-broadcastapp#23]
Is there any need for an advisor configuration screen? For example, if we have e-mail notifications (which might be necessary in the case of retrying errors), we might want to allow the advisors to set the e-mail to use.

**Comment by alecfrancesconi** [comment]
Yes - we should consider this type of functionality, perhaps as part of a larger admin functional set. There will be a need to manage the relationships between TAs, their account info and their state relationships.
